client/landing/stepper/declarative-flow/import-flow.ts

```diff
@@ -1,8 +1,6 @@
-import { isEnabled } from '@automattic/calypso-config';
-import { Design, isBlankCanvasDesign } from '@automattic/design-picker';
 import { IMPORT_FOCUSED_FLOW } from '@automattic/onboarding';
-import { useViewportMatch } from '@wordpress/compose';
 import { useDispatch, useSelect } from '@wordpress/data';
+import { SiteIntent } from 'calypso/../packages/data-stores/src/onboard';
```

Comment:
Should it be `@automattic/data-stores`?

Reply:
Nice catch, it was autocompleted, fixed, thanks!